pkg/ovs/openflow/ofctrl_builder.go

```diff
@@ -42,8 +43,27 @@ func (b *ofFlowBuilder) MatchTunMetadata(index int, data uint32) FlowBuilder {
 
 // MatchVLAN matches the VLAN VID. It holds the VLAN VID in its least significant 12 bits.
 func (b *ofFlowBuilder) MatchVLAN(nonVLAN bool, vlanID uint16, vlanMask *uint16) FlowBuilder {
-	// TODO(gran): correct matchStr
-	matchStr := fmt.Sprintf("dl_vlan=%d", vlanID)
```

Comment:
The current flow can work but it will be changed to `vlan_tci` after applied to OVS. Is this correct?

Reply:
Yes, when vlan ID value is 0, if mask is 0x1000, it will be changed to vlan_tci=0x1000/0x1000; if mask is 0x1fff, it will be dl_vlan=0